src/gui/render/graphics_shaders/quad_renderer_multisample.hpp

```diff
@@ -58,11 +58,6 @@ class QuadRendererMultisample : public QuadRenderer {
      */
     explicit QuadRendererMultisample(ShaderHandler shader_handler = ShaderHandler());
 
-    inline ~QuadRendererMultisample() {
```

Comment:
The dtor of `QuadRenderer` needs to be virtual for this to work.

Reply:
Why? Neither class is ever used as a pointer. Any way I will change it.

Reply:
Because, with inheritance, all virtual dtors are called on destruction, even if its not a ptr.